src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        int randomColor = new Random().nextInt(Colors.values().length);
+        return Colors.values()[randomColor].toString();
```

Comment:
Instead of using `toString()`, use `name()` to get the String representation of the enum constants as per the checklist.